client/src/components/App.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import axios from 'axios';
+
```

Comment:
I like how you separate the dependencies from the React components here.  I'm going to implement this into my project